### PR TITLE
Added missing integration/system tests for account_activities

### PR DESCRIPTION
**Pull request by maricavor**
Added additional fixtures for account_activities in order to test ransack search functionality

### CODE REVIEW

File: test/system/registrar_area/account_activities_test.rb [hunk 1]
```diff
@@ -25,4 +25,21 @@ def test_download_account_activity
...
+    visit registrar_account_activities_path
+
+    find('#q_activity_type_in').click
+    find(:option, "Renew").select_option
+    fill_in 'q_created_at_lteq', :with => '2021-12--'
+    find(:xpath, ".//button[./span[contains(@class, 'glyphicon-search')]]").click
+    
+    assert_text 'Description of activity renew_cash_1'
+  end
 end
```
Comment (yulgolem) [inline test/system/registrar_area/account_activities_test.rb:45]: Could you, please, add the empty string at the end of file?

**Comment by yulgolem** [comment]
.DS_Store is unnecessary here, move it to /.git/info/exclude or to .gitignore, please.

File: test/fixtures/account_activities.yml [hunk 2]
```diff
@@ -3,3 +3,33 @@ one:
   invoice: one
   bank_transaction: one
   created_at: <%= Time.zone.parse('2010-07-05 10:00') %>
+
+renew_cash_1:
```
Comment (yulgolem) [inline test/fixtures/account_activities.yml:7]: It's considered a good taste to name entities semantically, not just <something>_1 etc. Please, consider renaming.

File: test/integration/admin_area/account_activities_test.rb [hunk 3]
```diff
@@ -36,4 +36,29 @@ def test_download_account_activity
                   response.headers['Content-Disposition']
     assert_not_empty response.body
   end
+
+  def test_search_account_activity
+    account_activities(:one).update(description: "Description of activity one", 
+                                    sum: "123.00",
+                                    activity_type: "create",
+                                    created_at: Time.zone.parse('2021-07-05 10:00')
+                                   )
```
Comment (yulgolem) [inline test/integration/admin_area/account_activities_test.rb:45]: Could you, please, but ending bracket into the previous line?
Best practices are either 
```
(<some_args>,
<some_args>)
```
or
```
(
  <some_args>,
  <some_args>
)
```

**Review by yulgolem — CHANGES_REQUESTED** [review]
Overall all's good, but got some minor changes requested.

File: test/integration/admin_area/account_activities_test.rb [hunk 4]
```diff
@@ -36,4 +36,29 @@ def test_download_account_activity
...
+  
+    get admin_account_activities_path, params: { q: { account_registrar_id_in: [registrars(:bestnames).id, registrars(:goodnames).id], 
+                                                      activity_type_in: ['renew'], 
+                                                      created_at_gteq: '2021-09-25',
+                                                      created_at_lteq: '2021-11-'
+                                                    },
```
Comment (yulgolem) [inline test/integration/admin_area/account_activities_test.rb:51]: Same here - it's either
```
{key: value}
```
or
```
{
  key: value
}
```

File: test/system/registrar_area/account_activities_test.rb (hunk 1 above)
Comment (maricavor) [reply test/system/registrar_area/account_activities_test.rb]: Done!

**Review by yulgolem — APPROVED** [review]

File: Gemfile [hunk 5]
```diff
@@ -85,6 +85,7 @@ gem 'directo', github: 'internetee/directo', branch: 'master'
 
 group :development, :test do
   gem 'pry', '0.14.1'
+  gem 'rubocop'
```
Comment (OlegPhenomenon) [inline Gemfile:88]: It is better to add the rubocop gem in a separate branch, add general settings for the rubocop, and also with the consent of all other developers.

File: Gemfile [hunk 6]
```diff
@@ -25,7 +25,7 @@ gem 'validates_email_format_of', '1.6.3' # validates email against RFC 2822 and
 
 # 0.7.3 is the latest for Rails 4.2, however, it is absent on Rubygems server
 # https://github.com/huacnlee/rails-settings-cached/issues/165
-gem 'nokogiri', '~> 1.12.5'
+gem 'nokogiri', '~> 1.12', '>= 1.12.5'
```
Comment (OlegPhenomenon) [inline Gemfile:28]: Also, problems with updates must also be solved in a separate branch, it is not known when your branch will get into the master. Besides, the problem may be different. And so, for example, I may have an old version of nokogri and my PR will go to the master first of yours, and when it’s your turn to branch, a conflict may arise

**Review by OlegPhenomenon — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 5 above)
Comment (maricavor) [reply Gemfile:88]: I added rubocop only for development (and test) for manually checking files before commit.

File: Gemfile (hunk 6 above)
Comment (maricavor) [reply Gemfile:28]: Actually it is nothing to do with these settings, we already have the last version of nokogiri. 2 days ago ruby 3.1.0 released, but nikogiri cannot yet support this version

**Review by OlegPhenomenon — APPROVED** [review]

**Pull request closed without merging**